### ISSUE TITLE
[Question][Hw5] Abstract Semantics of Assign

### ISSUE THREAD

**Issue by ShinWonho** [prosyslab-classroom/cs524-program-analysis#42]
Hello, while implementing the HW5, I think there is a wrong description about abstract semantics of `Assign`

![image](https://user-images.githubusercontent.com/50018375/142721317-959c5d31-a405-483f-b7df-20de4ad7bec4.png)

Here is the abstract semantics of `Assign`.
The integer abstraction of the result of `eval` is wrapped to the value abstraction and, it is updated to the memory.
However, the result of `eval` is not an integer abstraction any more.

![image](https://user-images.githubusercontent.com/50018375/142721408-97fd8f34-8a32-43d3-bb38-b7f67e10f987.png)

As we can see, the result of eval is already a value abstraction.
Therefore, I think the abstract semantics should be just updating the value of x to the result of eval.

If there is any thing wrong, let me know.

Thank you!

**Comment by kangwoosukeq** [comment]
You're right. `eval` returns a value abstraction. I think it is our document problem. Sorry for the inconvenience.
